lib/src/controller.dart

```diff
@@ -1427,6 +1427,12 @@ class MapLibreMapController extends ChangeNotifier {
         .toList();
   }
 
+  /// Method to set style string
+  /// [styleString] -> It will take json, url, absolute path or asset path
+  Future<void> setStyle(String styleString) async {
```

Comment:
It's probably best to mention here platform specific differences like here:

https://github.com/maplibre/flutter-maplibre-gl/blob/44a09fc549dab25d70019da3228142afb72cab13/lib/src/maplibre_map.dart#L111-L120